tests/topotests/lib/common_config.py

```diff
@@ -789,7 +790,9 @@ def disable_v6_link_local(tgen, router, intf_name=None):
                     continue
 
                 cmd = "ip addr del {} dev {}".format(laddr, lname)
+                logger.info("%s", cmd)
                 router_list[router].run(cmd)
+                sleep(1)
```

Comment:
this sounds a bit scary :)

Reply:
I am not able to reproduce this locally, so i have put value as 1 sec to be on safer side. I have seen similar issue while applying ipv6 address for some of my scale automation and there 0.25 sec wait is enough.